test/testdata/lsp/code_actions/add_skeleton_for_abstract_method/existing_class_body.rb

```diff
@@ -0,0 +1,26 @@
+# typed: true
+# selective-apply-code-action: quickfix
```

Comment:
It's fine to have written these tests as quickfix tests, and keep them that way. But a tip of mine:

I would have written these tests as `*.autocorrects.exp` files. For example:

https://github.com/sorbet/sorbet/blob/0dbaf2c465314556b5acecec6ec63937f2b9e836/test/testdata/resolver/t_class.rb#L1-L8

https://github.com/sorbet/sorbet/blob/0dbaf2c465314556b5acecec6ec63937f2b9e836/test/testdata/resolver/t_class.rb.autocorrects.exp#L1-L10

This style of test is easier for changes like this, because if you ever make a change to the source file, you can simply regenerate the exp file automatically, either like this:

```bash
# regenerates all exp files
tools/scripts/update_testdata_exp.sh
```

or like this

```bash
# regenerates a single exp file, using the bazel-bin/main/sorbet that you already have built
tools/scripts/update_testdata_exp.sh --no-build test/testdata/.../foo.rb
```

As it stands, if you wanted to edit any of your existing tests, you have to repeat your edit manually in the `.rbedited` files, because we haven't automated regenerating those files.

Reply:
Thanks for the pro-tip! I converted to the `.autocorrect.exp` format in 8c5a72c. I had to introduce a new `test/testdata` subfolder for the "definition_validator" phase. I think I did so in a way that follows existing convention, but feedback on adjustments is welcome.